modal/app.py

```diff
@@ -288,12 +288,14 @@ async def lookup(
         environment_name: Optional[str] = None,
         create_if_missing: bool = False,
     ) -> "_App":
-        """Look up an app with a given name. When `create_if_missing` is true,
-        the app will be created if it doesn't exist.
+        """Look up an App with a given name, creating a new App if necessary.
+
+        Note that Apps created through this method will be in a deployed state
+        but will not have any Functions or Classes assocaited with them. This
```

Comment:
nit, comma before but

Reply:
I disagree! The text after "but" isn't an independent clause. :D